src/components/Visualization/Visualization.js

```diff
@@ -80,109 +81,120 @@ export const Visualization = ({
 
     return (
         <div className={styles.wrapper}>
-            <DataTable scrollHeight="500px" width="auto">
-                <DataTableHead>
-                    <DataTableRow>
-                        {data.headers.map((header, index) =>
-                            header ? (
-                                <DataTableColumnHeader
-                                    fixed
-                                    top="0"
-                                    key={header.name}
-                                    name={header.name}
-                                    onSortIconClick={({ name, direction }) =>
-                                        setSorting({
-                                            sortField: name,
-                                            sortDirection: direction,
-                                        })
-                                    }
-                                    sortDirection={
-                                        header.name === sortField
-                                            ? sortDirection
-                                            : 'default'
-                                    }
-                                    large={large}
-                                    small={small}
-                                    className={fontSizeClass}
-                                >
-                                    {header.column}
-                                </DataTableColumnHeader>
-                            ) : (
-                                <DataTableColumnHeader
-                                    fixed
-                                    top="0"
-                                    key={`undefined_${index}`} // FIXME this is due to pe not being present in headers, needs special handling
-                                    large={large}
-                                    small={small}
-                                    className={fontSizeClass}
-                                />
-                            )
-                        )}
-                    </DataTableRow>
-                </DataTableHead>
-                {/* https://jira.dhis2.org/browse/LIBS-278 */}
-                <DataTableBody loading={fetching}>
```

Comment:
I removed the `loading={fetching}` prop here, because we now have a custom loader UI.